tests/c/blockmap.c

```diff
@@ -1,3 +1,5 @@
+// # The function __yktrace_hwt_mapper_blockmap_len is compile-time guarded for hwt only.
+// ignore-if: test $YKB_TRACER == "swt"
```

Comment:
I think this should be `test $YKB_TRACER != "hwt"`.

Reply:
Yes, you are right
Actually, it should be:
```
test "$YKB_TRACER" != "hwt" && test "$YKB_TRACER" != ""
```
Since by default, `YKB_TRACER` is not set

Reply:
updated 👉 2c494bfc79f1a4741475c3464ab9f2885472aec3

Reply:
We should always be setting a variable so we know which tracer we're setting. I agree that we don't do that consistently right now.

Reply:
Ok, I can set it to be `test "$YKB_TRACER" != "hwt"` and update CI tests accordingly

Reply:
https://github.com/ykjit/yk/pull/953 might give you pointers as to where we want to move (but we're not all the way there yet).